scalameta/contrib/src/main/scala/scala/meta/contrib/Equal.scala

```diff
@@ -0,0 +1,12 @@
+package scala.meta.contrib
+
+import scala.meta.prettyprinters.Show
```

Comment:
I would say the typeclass `Show` should be in a separate package. The instances can still be defined in prettyprinters.

It might make sense to have:
scala.meta.typeclass
and 
scala.meta.contrib.typeclass

as packages. I'm still undecided though.